Formula/osslsigncode.rb

```diff
@@ -18,19 +18,18 @@ class Osslsigncode < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "libgsf"
   depends_on "openssl@1.1"
 
   uses_from_macos "curl"
 
   def install
-    system "./autogen.sh"
-    system "./configure", "--with-gsf", "--prefix=#{prefix}"
+    system "./bootstrap"
+    system "./configure", "--prefix=#{prefix}"
```

Comment:
We have this now right?
```suggestion
    system "./configure", *std_configure_args
```

Reply:
`--disable-debug` is not recognized by the configure script but it builds fine anyway, so we can roll with it.